content/source/docs/cloud/agents/index.html.md

```diff
@@ -195,7 +206,7 @@ You may choose to run multiple agents within your network, up to the organizatio
 
 #### Resilience
 
-If an agent terminates abnormally, any running jobs can be restarted where they left off by resubmitting them. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
+It is possible that an agent process could be terminated unexpectedly (due to killing the process forcefully, power cycling the host machine, etc.). We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
```

Comment:
```suggestion
It is possible that an agent process could be terminated unexpectedly (due to killing the process forcefully, power cycling the host machine, etc.). We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error. If this happens, any running jobs can be restarted where they left off by resubmitting them.
```

This seemed like an important bit of info that was no longer included, otherwise LGTM!

Reply:
That is the part that isn't true. You can't resume work that was in-flight when an agent is terminated unexpectedely. Runs will need to be canceled and enqueued again, which is a new run, not a restart. This is actually the specific bit that felt misleading to me. Does that make sense?

Reply:
Aha, yes! That's info I was missing from the resolution yesterday. I should've known there was a reason you left it out. Makes sense to me, everything else looks good.